File: level52/homework/codwars4.py
#https://www.codewars.com/kata/57356c55867b9b7a60000bd7/train/python
def basic_op(operator, value1, value2):
    match operator:
        case '+':
            return value1 + value2
        case '-':
            return value1 - value2
        case '*':
            return value1 * value2
        case '/':
            return value1 / value2
    

File: level72/homework/codwars4.js
//https://www.codewars.com/kata/5ab6538b379d20ad880000ab/train/javascript
const areaOrPerimeter = function(l , w) {
  if (w == l) {
    return w * l;
  }
    else {
      return 2*l+2*w;
    }
};